client/package-lock.json

```diff
@@ -1,264 +1,8 @@
 {
 	"name": "rescript-language-client",
 	"version": "0.0.1",
-	"lockfileVersion": 2,
+	"lockfileVersion": 1,
```

Comment:
Humm I guess this should not be in this diff? Also the team's using a newer version of npm, thus the lockfile version difference

Reply:
You're right. Should restore the lock-file. :-)